src/image_intel.cpp

```diff
@@ -108,15 +108,21 @@ static void unmountFilesystem(const std::string& filesystem)
  */
 static void releaseFlashDrive()
 {
-    Tracer tracer("Release flash drive");
+    const std::vector<std::string> units = {
+        "rotate-event-logs.timer", "rotate-event-logs.service",
+        "logrotate.timer",         "logrotate.service",
+        "nv-sync.service",
+    };
 
-    stopUnit("rotate-event-logs.timer");
-    stopUnit("rotate-event-logs.service");
-    stopUnit("logrotate.timer");
-    stopUnit("logrotate.service");
-    stopUnit("nv-sync.service");
```

Comment:
All good to me, but do we have a way to define at runtime a set of services that are blocking I/O?
I'm pretty sure in future we get a new one which is blocking IO too. 
May be we can trace FDs or capture a currently executed IO operations by take a look at the /proc for get list of services wich blocks partitions? Just propose to think :)

Reply:
Unfortunately, there are no enough clear indicators which software may impede to unmount partitions. 
At least, I don't see how can it be done automatically at run time.